qutebrowser/completion/completionwidget.py

```diff
@@ -203,6 +203,10 @@ def set_model(self, model):
             sel_model.deleteLater()
         for i in range(model.rowCount()):
             self.expand(model.index(i, 0))
+
+        if model.column_widths:
+            self.column_widths = model.column_widths
```

Comment:
I'd prefer doing `model.source.column_widths` here and not set anything on the `CompletionFilterModel`.